### DISCUSSION TITLE
Move to GitHub organization?

### DISCUSSION THREAD

**Discussion by rickardp** [rickardp/bitsandbytes#2] [Ideas]
It's easier to manage the community effort if we move to a specific organization rather than my personal one. Does anyone have a good name for it? 🙂

 @phdykd @eusip @matthewdouglas @dosier @marcothedeveloper123 @deep-pipeline

**Comment by dosier** [comment]
Maybe something fun that does a wordplay on biting, like BitingBytes, or maybe something with teeth or mouth or taste. Mmmmm, not sure :D. Will give it some more thought.

**Reply by unixwzrd** [reply]
Checkout my wordplay, I like it, but then that's just shameless self-promotion.  However something of interest, I've spent most of the day re-building my entire set of binaries and python venv's for oobabooga and it appears there is a bug in there somewhere or several places where it's not handling GPU correctly, I put together some bogus tensor python/torch to tress test my system and indeed I did see my GPU get pegged!  So my build stuff in definitely working correctly, no I'm going in and fixing the code.

Sorry to hijack your response, but just noticed your emphasis on words on plays.

Cheers!

**Comment by phdykd** [comment]
DigitalDoughnuts

**Comment by unixwzrd** [comment]
NybblesAndBits

Oh, yes, and I want this as well.  Don't know much about GPU's, but maybe I can help somehow.  Testing, installation, documentation, just give a shout.

MBP M2 Max 12 CPU, 38 GPU, 96GB RAM

**Comment by rickardp** [comment]
Closing this discussion as the official repo has now become more active and supports the portability initiative started here and elsewhere. Please continue discussion in https://github.com/TimDettmers/bitsandbytes/discussions/990
